Sources/StreamChatUI/CommonViews/ContainerStackView.swift

```diff
@@ -311,11 +311,6 @@ public class ContainerStackView: UIView {
             : safeAreaLayoutGuide
     }
 
-    public func addArrangedSubview(_ builder: () -> UIView) {
-        addArrangedSubview(builder())
-    }
-
-
```

Comment:
This reminds me, I think it would make sense to add
```
public func addArrangedSubviews(_ views: [UIView])
public func removeAllArrangedSubviews()
```

Reply:
Yes the `public func addArrangedSubviews(_ views: [UIView])` we already have, but not the `removeAllArrangedSubviews()`. It makes sense to add it 👍 And maybe adding a method the replace all the arrangedSubviews with the provided array? 🤔

Reply:
✅  Done!